docs/navigation.rb

```diff
@@ -243,30 +243,35 @@ def without_prefix(prefix)
         nav_order: 2,
         children: [
           {
-            path: 'Using-Metasploit.md',
-            title: 'Running modules',
+            path: 'Metasploit-Filesystem-Explained.md',
+            title: "Metasploit's Filesystem Explained",
```

Comment:
I'm thinking as an average user, the internals of Metasploit's directory structure might be redundant information 🤔 

I'm thinking this might be better suited either living in the developer section, or potentially we should just have `README.md` files in each of the directories that you would like to add documentation to? The rationale being, if a user/developer is looking at folders there's a higher chance of discovering that documentation as they explore their filesystem, instead of them having to know that the information exists elsewhere in a docs site

Reply:
I'm open to either and was also thinking of if the README.md approach might be better.

I took cues on this from Offensive Security's Metasploit Unleashed course which has a page dedicated to doing a similar overview of Metasploit's file structure in case that helps.

Reply:
@adfoster-r7 Happy to adjust this to take this approach if people want to go that way. In the meantime I've taken the advice solicited so far and been using that to adjust this PR so if we do decide we want to go with the README approach it should be easy to just copy the info from here into the existing file or a new one if it doesn't exist.

Reply:
Do we have an update on this? Otherwise this seems to be in limbo at the moment waiting for a decision on this point.

Reply:
IMO this information most likely either lives in the developer section, or as separate README.md files in the each of the corresponding folders